docs/api-reliability.md

```
@@ -0,0 +1,460 @@
+
+### Gluetun API
+
+The Gluetun API now includes automatic retry logic with configurable delays:
+
+**Configuration Variables:**
+- `GLUETUN_API_RETRY_COUNT` - Number of retry attempts (default: 3)
+- `GLUETUN_API_RETRY_DELAY` - Delay between retries in seconds (default: 2)
+- `GLUETUN_API_TIMEOUT` - Request timeout in seconds (default: 8)
+
+**Example:**
+```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)